### ISSUE TITLE
[a11y] On group field questions, the question isn't read by screen reader when using tabbing to navigate

### ISSUE THREAD

**Issue by melanie-exygy** [civiform/civiform#10221]
**Describe the bug**
On address, name or date questions, when using the tab key or form navigation, the screen reader only reads the field label and doesn't read the question.  This happens both when in an error state and when not in an error state.  This could be confusing because the user might not know what specific name or address the form is asking for.

For example, on a name question, if there is a validation error, the screen reader will say "First name, required invalid text".  The user may not know who's first name they should enter.  Is it theirs, their spouse's, or the first name of the contractor working on their roof?

**To Reproduce**

1. Start applying to the Comprehensive Sample Program.
2. Turn the screen reader on.
4. Tab to one of the fields on the address question.
5. Note the the screen reader says the field name (i.e "Zip code"), but not the question itself ("What is your home address?").

**Expected behavior**
The screen reader should read the question and the name of the field, along with the help text and error description.

**Desktop (please complete the following information):**

- OS: Mac
- Browser: Chrome
- Screen reader: VoiceOver
